tools/aa/commands/InitCommand.java

```diff
@@ -95,11 +97,41 @@ public boolean run(String[] args) {
     return true;
   }
 
+  private boolean processArgs(String[] args) {
+    boolean isAaSetUp = args[0].equals("aa");
+    if (isAaSetUp) {
+      switch (args.length) {
+        case 1:
+          System.err.println(
+              RED_ERROR
+                  + "Invalid usage. \n"
+                  + "Please use \"aa init <base_path>\" command to init a base folder.");
+          return false;
+        case 2:
+          System.err.println(RED_ERROR + "Missing base_path");
```

Comment:
I think you should end in `ANSI_RESET` to prevent the next text to be red too.

Reply:
Do you mean `System.err.println(RED_ERROR + "Missing base_path" + ANSI_RESET)`?

I don't see any diff beetwen `(RED_ERROR + "Missing base_path" + ANSI_RESET)` and `(RED_ERROR + "Missing base_path").` In both cases, the text looks like:
<red>ERROR:<red> <usual>Missing base_path<usual>
<usual>some text<usual>